native_words.asm

```diff
@@ -2116,7 +2116,7 @@ _compile_check:
                 and #NN
                 beq _check_size_limit
 
-                jmp _compile_as_jump    ; too far for BRA
+                jmp compile_as_jump    ; too far for BRA
```

Comment:
creating a new entry point meant globalizing some other labels.  not sure if there's a better way to do that

Reply:
I don't think there is a better way to do that.  I've actually considered making all of Tali's labels global and naming all of the anonymous labels in order to make it easier to switch assemblers, but then I remember how many _done labels there are :)